### ISSUE TITLE
Sass for content scripts does not get processed

### ISSUE THREAD

**Issue by Bauke** [aklinker1/vite-plugin-web-extension#19]
Hi, I'm trying to convert a Parcel-based WebExtension to Vite and I have a small question.

I have a content scripts section that looks like this:

```json
"content_scripts": [
  {
    "matches": [
      "*://tildes.net/*"
    ],
    "run_at": "document_end",
    "css": [
      "scss/scripts.scss"
    ],
    "js": [
      "content-scripts.ts"
    ]
  }
],
```

And while the TS gets built as expected, the Sass is copied over as-is which wasn't what I expected. I figured out I can change the CSS definition to `"generated:style.css"` and then add `import './scss/scripts.scss';` to my `content-scripts.ts` file so the Sass does get processed, but it's not exactly ideal.

So my question is, is this how I should be doing it or is there a better way?

**Comment by aklinker1** [comment]
Hmm, nope that is correct. You shouldn't have to do the `generated:` to get it working. I'll get a fix out for this when I have time, hopefully the next few days

**Comment by aklinker1** [comment]
Just published [v1.1.3](https://github.com/aklinker1/vite-plugin-web-extension/releases/tag/v1.1.3), feel free to re-open this issue it doesn't work for you!

**Comment by Bauke** [comment]
Seems to work great, thanks for the quick fix!
